quill-sql/src/test/scala/io/getquill/context/sql/norm/ExpandNestedQueriesSpec.scala

```diff
@@ -27,7 +27,7 @@ class ExpandNestedQueriesSpec extends Spec {
       } yield b.i).take(10)
     }
     testContext.run(q).string mustEqual
-      "SELECT x.* FROM (SELECT b.i FROM TestEntity a, TestEntity2 b) x LIMIT 10"
+      "SELECT b.i FROM TestEntity a, TestEntity2 b LIMIT 10"
```

Comment:
these test cases are about of correct nested queries, but take/drop shouldn't compiled in nested query (my current fix). 
I have opened an issue #782, manual `nested` causes stack overflow